iwyu.cc

```diff
@@ -2604,6 +2602,17 @@ class IwyuBaseAstVisitor : public BaseAstVisitor<Derived> {
     return retval;
   }
 
+ private:
```

Comment:
Oh, I see what happened now; you moved `GetUsingDeclarationOf` from private to protected, makes sense, thanks.